api-testing/multicloud-api/README.md

```diff
@@ -0,0 +1,27 @@
+# Multi cloud API Automation
+
+**Introduction**
+
+This framework has developed in [JUNIT](https://junit.org/) using [Gradle](https://gradle.org/) Build tool.
+Junit framework details [Link](https://www.tutorialspoint.com/junit/index.htm)
+
+**Why this?**
+
+Junit is widely used testing framework along with Java Programming Language. You can use this automation framework for both unit testing and UI testing.
```

Comment:
Why dont we give an introduction to say about this framework?
Currently the framework is for API testing for multicloud -- that details...

It looks something generic.

Reply:
Can i use like API Testing Automation Framework?

Reply:
Why not? Please mention it is meant for API (REST) testing and currently added for multicloud project of soda foundation. Later it could be reused for other projects...etc...Give all the details about the framework

Reply:
I have changed. Please review.